src/orquestra/sdk/_ray/_ray_logs.py

```diff
@@ -41,6 +42,72 @@ class WFLog(pydantic.BaseModel):
     task_run_id: t.Optional[TaskRunId]
 
 
+# region: CE system logs
+
+
+class SystemLogSourceType(Enum):
+    """Types of sources that can emit system logs."""
+
+    RAY_HEAD_NODE = "RAY_HEAD_NODE"
+    RAY_WORKER_NODE = "RAY_WORKER_NODE"
+    K8S_EVENT = "K8S_EVENT"
+
+
+class K8sEventLogContainerInfoStatus(pydantic.BaseModel):
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Kubernetes logs are not created by Ray, do they belong here?